aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -2646,13 +2646,19 @@ pub(crate) fn is_account_init_for_sponsored_transaction(
     features: &Features,
     resolver: &impl AptosMoveResolver,
 ) -> VMResult<bool> {
+    let account_resource_struct_tag = AccountResource::struct_tag();
```

Comment:
Maybe more of a personal taste, but I feel like `AccountResource::struct_tag()` is not really longer than `account_resource_struct_tag` so this does not seem to improve readability.

Reply:
Removed because of duplication, but can inline back, no preference here from my side 👍